meson.options

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+option(
```

Comment:
Please add a comment to clarify that the choice must be the file name (without extension) from the config/data_sync_list directory.

Reply:
Done